mailstream/mailstream.php

```diff
@@ -444,19 +439,7 @@ function mailstream_html_wrap(&$text)
  */
 function mailstream_convert_table_entries()
 {
-	$query = <<< EOT
-SELECT
-  `message-id`,
-  `uri`,
-  `uid`,
-  `contact-id`
-FROM
-   `mailstream_item`
-WHERE
-  `mailstream_item`.`completed` IS NULL
-
-EOT;
-	$ms_item_ids = q($query);
+	$ms_item_ids = DBA::selectToArray('mailstream_item', ['message-id', 'uri', 'uid', 'contact-id'], ["`mailstream_item`.`completed` IS NULL"]);
```

Comment:
```suggestion
	$ms_item_ids = DBA::selectToArray('mailstream_item', ['uid'], ['message-id', 'uri', 'uid', 'contact-id'], ["`mailstream_item`.`completed` IS NULL"]);
```

Reply:
No, that's okay here. The uid mustn't added here.